packages/components/ng-ovh-feature-flipping/README.md

```diff
@@ -0,0 +1,34 @@
+# ng-ovh-feature-flipping
```

Comment:
Is it global to ovh or just manager ?

Reply:
This is global.
Rules in the 2API can be declared by order team and for order for example.